src/gui/csvImport/CsvImportWidget.ui

```diff
@@ -498,13 +409,7 @@
   </customwidget>
  </customwidgets>
  <tabstops>
-  <tabstop>comboBoxCodec</tabstop>
-  <tabstop>comboBoxTextQualifier</tabstop>
-  <tabstop>comboBoxFieldSeparator</tabstop>
-  <tabstop>comboBoxComment</tabstop>
-  <tabstop>checkBoxFieldNames</tabstop>
   <tabstop>spinBoxSkip</tabstop>
-  <tabstop>checkBoxBackslash</tabstop>
   <tabstop>tableViewFields</tabstop>
```

Comment:
Any reason why you removed these?

Reply:
Go go crappy Qt Designer! They should not have been removed, good catch.

Reply:
Did you fix it? I am still seeing the deletions.